docs/user/index.rst

```diff
@@ -79,7 +79,6 @@ and some of the core features of Read the Docs.
   :doc:`/downloadable-documentation` |
   :doc:`/hosting` |
   :doc:`/server-side-search` |
-  :doc:`/analytics` |
```

Comment:
I agree we need a feature overview page before this gets merged. Figuring out what's on the page seems a bit tricky if it doesn't include how to enable/use it..

Reply:
The lack of a good feature description for Traffic Analysis is already noted. So I wanted to move forwards with the small refactors firstly before starting to write new contents.

But it seems I'm outnumbered now since you've also joined the other camp :joy:

Reply:
I mean, this is actively making the docs worse for users to discover our features, so we should do the proper change.

Reply:
We are in a process, each incremental step holds a direction towards Diátaxis. I don't agree that _when_ we re-label a 90% How To guide as a How To Guide, things are worsened. I think they improve. We note for the immediate future that we need a feature reference authored.

I recommend this: https://diataxis.fr/how-to-use-diataxis/

Especially "Work one step at a time" and "Just do something".

Reply:
Sure, the relabeling isn't the issue, it's the fact that we no longer list a primary feature in our feature list.

Reply:
Of course I want our features represented. I'm stressing that there is no current content for such a feature page because the existing content was a How-To. Therefore, I'm writing down imminent subsequent steps.

The plan was to start by making small steps to move contents into their right places. I think that's what I'm doing with this PR. I have no problem at all with creating a feature page, too. But then we are changing the original plan.

Reply:
The whole plan is to not have a plan :) A goal also needs to be working incrementally and not making the docs worse in the meantime. Instead of debating this, you should just write the feature page, it would take the same amount of time..